code/modules/mob/living/simple_animal/friendly/farm_animals.dm

```diff
@@ -376,6 +379,10 @@
 	gold_core_spawnable = FRIENDLY_SPAWN
 	chat_color = "#FFDC9B"
 
+/mob/living/simple_animal/chicken/turkey/Initialize(mapload)
+	. = ..()
+	src.AddComponent(/datum/component/waddling)
```

Comment:
I don't think this is actually needed.
The chicken parent call is already adding the waddling component

Reply:
Fair point, fixed!